devtools/maven/src/main/resources/create-extension-templates/deployment-pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>[=groupId]</groupId>
+        <artifactId>[=artifactId]-parent</artifactId>
+        <version>[=version]</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>[=artifactId]-deployment</artifactId>
+[#if nameBase?? ]    <name>[=namePrefix][=nameBase][=nameSegmentDelimiter]Deployment</name>
+[/#if]
+
+    <dependencies>
```

Comment:
The deployment artifact should depend on the runtime artifact. This must be documented in the guide.

Reply:
+1 Actually, I thought, I had it there. Thanks for spotting this!

Reply:
Fixed in 344226776dbd42ba63a4c2163d3ba9955cc232cf